lib/open_mail_app.dart

```diff
@@ -13,6 +13,7 @@ class OpenMailApp {
   OpenMailApp._();
 
   static const MethodChannel _channel = const MethodChannel('open_mail_app');
+  static List<String> _filterList = <String>[];
```

Comment:
I feel like `_blockList` makes more straightforward but I am okay with `_filterList` as well.

Reply:
So, the term `blacklist`, as constrasted with `whitelist`, which has been used historically, has some problematic racial connotations. `blocklist` makes some sense, but not necessarily any more so than does `filterlist`.

Reply:
blocklist and allowlist I have heard are gaining traction as replacements. But I have no strong opinion on changing it for this case.